vignettes/Demography.Rmd

```diff
@@ -41,116 +55,186 @@ simparams <- get_parameters(
   )
 )
 
+# Use set_equilibrium to tune the human and mosquito populations to those required for the defined EIR
 simparams <- set_equilibrium(simparams, starting_EIR)
+
 ```
 
-## Custom demography
+## **Custom Demographic Parameterisation**
 
-We can set a custom demography:
+Next, we'll use the the in-built `set_demography()` function to specify human death rates by age group. This function accepts as inputs a parameter list to update, a vector of the age groups (in days) for which we specify death rates, a vector of time steps in which we want the death rates to be updated, and a matrix containing the death rates for each age group in each timestep. The `set_demography()` function appends these to the parameter list and updates the `custom_demography` parameter to `TRUE`.
 
 ```{r}
-demography_params <- simparams
 
-# Set a flat demography
-ages <- round(c(
-  .083333, 1, 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80,
-  85, 90, 95, 200) * year)
+# Copy the simulation parameters as demography parameters:
+dem_params <- simparams
+
+# We can set our own custom demography:
+ages <- round(c(0.083333, 1, 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 200) * year)
 
+# Set deathrates for each age group (looks like by taking annual values and dividing by 365:
 deathrates <- c(
   .4014834, .0583379, .0380348, .0395061, .0347255, .0240849, .0300902,
   .0357914, .0443123, .0604932, .0466799, .0426199, .0268332, .049361,
   .0234852, .0988317, .046755, .1638875, .1148753, .3409079, .2239224,
   .8338688) / 365
 
-demography_params <- set_demography(
-  demography_params,
+# Pass these custom ages and death rates through the in-built set_demography function 
+dem_params <- set_demography(
+  dem_params,
   agegroups = ages,
   timesteps = 0,
```

Comment:
could just add a little comment here to say that you are starting the custom demography at the beginning of the model run

Reply:
I have updated the comment to:

"# Update the parameter list with the custom ages and death rates through the in-built
# set_demography() function, instructing the custom demography to be implemented at the
# beginning of the model run (timestep = 0):"

and included the following in the textual explanation:

"Here, we instruct the model to implement the changes to the demographic parameters at the beginning of the simulation (`timesteps = 0`)."